src/app/contexts/ToggleContext/index.test.jsx

```diff
@@ -20,16 +25,67 @@ const mockToggleValues = {
   },
 };
 
+const mockToggleValuesRemoteFeatureToglles = {
+  local: {
+    remoteFeatureToggles: {
+      enabled: true,
+    },
+  },
+};
+
+const mockTogglesResponse = {
+  toggles: {
+    toggleName: {
+      enabled: true,
+      value: '',
+    },
+  },
+};
+
 jest.mock('#lib/config/toggles', () => mockToggleValues);
+jest.mock('./utils/constructTogglesEndpoint', () =>
+  jest.fn().mockReturnValue('test.com'),
+);
+
+beforeEach(() => {
+  process.env.SIMORGH_APP_ENV = 'local';
+  global.fetch = fetch;
+});
+
+afterEach(() => {
+  fetchMock.restore();
+});
 
 // Require after mock to allow mocking of JS object
 const { ToggleContext, ToggleContextProvider } = require('./index');
 
-describe('ToggleContext', () => {
+describe('ToggleContext without remote feature toggles', () => {
+  // fetchMock.mock('test.com', mockTogglesResponse);
+
+  shouldMatchSnapshot(
+    `should provide a toggles object`,
+    <ToggleContextProvider service="mundo">
+      <ComponentUsingContext context={ToggleContext} />
+    </ToggleContextProvider>,
+  );
+});
+
+describe('ToggleContext with remote feature toggles', () => {
+  jest.mock('#lib/config/toggles', () => mockToggleValuesRemoteFeatureToglles);
+  fetchMock.mock('test.com', mockTogglesResponse);
+
   shouldMatchSnapshot(
```

Comment:
what if `ComponentUsingContext` actually used the toggles context to toggle a mock ad (like the grey box component) on or off and we checked it was in the document or not?

Reply:
We have written a new integration test doing just this now :)